zict/buffer.py

```diff
@@ -165,3 +178,25 @@ def flush(self) -> None:
 
     def close(self) -> None:
         close(self.fast, self.slow)
+
+
+class BufferItemsView(ItemsView[KT, VT]):
```

Comment:
Should we also override `__contains__` in `BufferItemsView` to avoid changing the LRU?

Reply:
Thanks for the explanation; now it makes sense to me. I had assumed we wanted this to be a general property of the views.

>  If you're testing if your buffer contains a specific key/value _pair_, and that causes the value to be unevicted from slow, then I don't see why it should bypass the principle of temporal locality.

Sounds good to me!